### PR TITLE
MultiMC and Hyperium 1.3 support

### PR DESCRIPTION
**Pull request by ghost**
UI done by @Systemless03

### CODE REVIEW

**Comment by ghost** [comment]
queued for 1.3

**Review by Cubxity — COMMENTED** [review]
Has this been tested?

File: src/main/kotlin/cc/hyperium/installer/ui/stages/AddonsSelectionStage.kt [hunk 1]
```diff
@@ -36,7 +36,7 @@ import java.net.URI
 import java.net.URL
 import java.net.URLEncoder
 
-class AddonsSelectionStage() : View() {
```
Comment (emmaexe) [inline src/main/kotlin/cc/hyperium/installer/ui/stages/AddonsSelectionStage.kt]: Here it is! Found it for u.

Comment (ghost) [reply src/main/kotlin/cc/hyperium/installer/ui/stages/AddonsSelectionStage.kt]: ?

**Pull request merged**